src/lib/config.ts

```diff
@@ -1,16 +1,16 @@
-require("dotenv").config();
+import cfenv from "cfenv";
+import dotenv from "dotenv";
 
-const cfenv = require("cfenv");
-const appEnv = cfenv.getAppEnv();
-const serviceConfig = {};
+dotenv.config();
 
-if (!appEnv.isLocal) {
-  serviceConfig.coreBackAPIUrl = appEnv.getServiceURL("core-back-api");
-}
+const appEnv = cfenv.getAppEnv();
+const coreBackApiUrl = appEnv.isLocal
+  ? undefined
+  : appEnv.getServiceURL("core-back-api");
 
-module.exports = {
+export default {
```

Comment:
This is a slight change (and has caused all the JS imports to need updating), but feels easier to manage than exporting each config value individually.

Also means the imports (once using TS) are cleaner - as you can just `import config from "./config"` and then access things on that object.